doc/install/ceph-deploy/quick-start-preflight.rst

```diff
@@ -28,7 +28,7 @@ For Debian and Ubuntu distributions, perform the following steps:
 
 #. Add the Ceph packages to your repository. Use the command below and
    replace ``{ceph-stable-release}`` with a stable Ceph release (e.g.,
-   ``luminous``.)  For example::
+   :substitution-code:`|stable-release|`.)  For example::
```

Comment:
I like the idea of `stable-release` but perhaps we shouldn't be using octopus an example with using ceph-deploy.

Reply:
i changed it back. but why? because ceph-deploy does not work with octopus?